### PR TITLE
New buttons with dark images

### PR DESCRIPTION
**Pull request by dparikh**
b/116257803
Existing images: https://screenshot.googleplex.com/ALWp08AFsqN.png
Modified images: https://screenshot.googleplex.com/vY8RRBooneb.png

### CODE REVIEW

File: assets/swg-button.css [hunk 1]
```diff
@@ -61,7 +61,7 @@
...
...uNTE3ODQxNiwxMi45MDE1NTQ0IEM0OC41MTc4NDE2LDE0Ljk0NzAzNTEgNDkuOTczMDIsMTYuNDM0NjU3NSA1MS43MTU0MDQ3LDE2LjQzNDY1NzUgQzUzLjQyOTA2ODgsMTYuNDM0NjU3NSA1NC43Nzg5MzgyLDE0Ljk0NzAzNTEgNTQuNzc4OTM4MiwxMi45MDE1NTQ0IFoiIGlkPSJTaGFwZSI+PC9wYXRoPgogICAgICAgICAgICA8cG9seWdvbiBpZD0iU2hhcGUiIHBvaW50cz0iNjMgMSA2MyAxOSA2MCAxOSA2MCAxIj48L3BvbHlnb24+CiAgICAgICAgICAgIDxwYXRoIGQ9Ik03Mi44NDQxMjAzLDE0Ljk3MzI4ODggTDc0Ljg4OTY5OTIsMTYuMzM1NTU5MyBDNzQuMjI3ODk0MywxNy4zMTcxOTUzIDcyLjYzMzU0NiwxOSA2OS44NzYwMjU1LDE5IEM2Ni40NTY3MDAxLDE5IDY0LDE2LjM2NTYwOTMgNjQsMTMgQzY0LDkuNDM0MDU2NzYgNjYuNDg2NzgyMSw3IDY5LjU4NTIzMjUsNyBDNzIuNzAzNzM3NSw3IDc0LjIyNzg5NDMsOS40NzQxMjM1NCA3NC43MjkyNjE2LDEwLjgxNjM2MDYgTDc1LDExLjQ5NzQ5NTggTDY2Ljk2ODA5NDgsMTQuODIzMDM4NCBDNjcuNTc5NzYzLDE2LjAyNTA0MTcgNjguNTQyMzg4MywxNi42NDYwNzY4IDY5Ljg4NjA1MjksMTYuNjQ2MDc2OCBDNzEuMjI5NzE3NCwxNi42NDYwNzY4IDcyLjE2MjI2MDcsMTUuOTc0OTU4MyA3Mi44NDQxMjAzLDE0Ljk3MzI4ODggWiBNNjYuNTM2OTE4OSwxMi44MDk2ODI4IEw3MS45MTE1NzcsMTAuNTg1OTc2NiBDNzEuNjEwNzU2Niw5LjgzNDcyNDU0IDcwLjcyODM1LDkuMzEzODU2NDMgNjkuNjc1NDc4Niw5LjMxMzg1NjQzIEM2OC4zMzE4MTQsOS4zMTM4NTY0MyA2Ni40NjY3Mjc0LDEwLjQ5NTgyNjQgNjYuNTM2OTE4OSwxMi44MDk2ODI4IFoiIGlkPSJTaGFwZSI+PC9wYXRoPgogICAgICAgIDwvZz4KICAgIDwvZz4KPC9zdmc+');
```
Comment (dvoytenko) [inline assets/swg-button.css]: Is this change necessary? It looks like it significantly adds to the size?

Comment (dparikh) [reply assets/swg-button.css]: Updated the encoding.
